katcp/test/test_testutils.py

```diff
@@ -202,3 +203,26 @@ def test_ioloop_hogging(self):
         DUT(1)
         self.io_loop.add_callback(DUT, 2)
         yield DUT.assert_wait_call_count(2, timeout=0.1)
+
+
+class test_AssertNoMemoryLeaks(unittest.TestCase):
+
+    def test_non_leaky_object(self):
+
+        class NonLeaky(object):
+            pass
+
+        with testutils.assert_no_memory_leaks():
+            obj = NonLeaky()
+            obj = None  # noqa: F841
```

Comment:
You could replace both lines with just `NonLeaky()`.

Reply:
Doh!  That's a better idea.